### ISSUE TITLE
problème "nil"

### ISSUE THREAD

**Issue by jaykayone** [Geoportail-Luxembourg/qgis-pag-plugin#114]
Le GML exporté contient des tags de type nil=True
```
<CONST_A_CONS_POINT gml:id="CONST_A_CONS_POINT.0">
**<CATEGORIE xsi:nil="true"/>**
<NOM_FICHIER>014_PE_CONST_A_CONS</NOM_FICHIER>
<GEOMETRIE>
<gml:Point gml:id="CONST_A_CONS_POINT.geom.0" srsName="urn:ogc:def:crs:EPSG::2169">
<gml:pos>103496.8466 79178.7021995467</gml:pos>
</gml:Point>
</GEOMETRIE>
</CONST_A_CONS_POINT>
```
Notre checker n'accepte pas cette syntaxe.
Dans l'ancien export, ces tags étaient omis

**Comment by frankbroniewski** [comment]
Hi,

I believe this is introduced through GDAL/OGR which is used to create the GML first hand: [[1](https://github.com/Geoportail-Luxembourg/qgis-pag-plugin/blob/master/widgets/export_gml/export_gml.py#L122)]

I've not found so far a OGR GML driver creation option to change that behavior. According to the W3C XML specification, NULL <> empty [[2](https://www.w3.org/TR/xmlschema-1/#xsi_nil)], so the generated XML is valid according to the specification, but this is probably not very helpful if your validator does not like this.

I think a hook to adapt this might be in [[3](https://github.com/Geoportail-Luxembourg/qgis-pag-plugin/blob/master/widgets/import_data/importer.py#L221)] where an explicit check for NULL values is done through the plugin. A possible solution might be to set the attribute not to NULL, but to an empty string (in case of string values of course).

Anyway, just my 2c, since I'll need to use the plugin in the foreseeable future a couple of times again

**Comment by arx-it** [comment]
Bonjour,

Doit-on effectuer une correction du plugin alors? Ou le checker sera adapté pour supporter la spécification W3C?

**Comment by jaykayone** [comment]
il faut une correction du plugin

**Comment by arx-it** [comment]
Fixed in 081c3d401db1a19e384702bc08e4979f3ac24e91
